scripts/mapCoverage.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+/**
+ * Because we have a build step in jest, we collect coverage from both
+ * `packages/src/*` and `package/build/*` files during our test run. They are
+ * practically the same files. This script will merge them together
+ * and write the final coverage report.
+ */
```

Comment:
I'm not sure I understand this. How are we collecting coverage for the build files? They should be ignored in Jest itself and be used to run the src files only, so they aren't instrumented themselves.